docs/reference/lambda-support.md

```diff
@@ -0,0 +1,261 @@
+---
+mapped_pages:
+  - https://www.elastic.co/guide/en/apm/agent/python/current/lambda-support.html
+sub:
+  apm-lambda-ext-v: ver-1-5-7
+  apm-python-v: ver-6-23-0
+---
+
+# Monitoring AWS Lambda Python Functions [lambda-support]
+
+The Python APM Agent can be used with AWS Lambda to monitor the execution of your AWS Lambda functions.
+
+:::{note}
+The Centralized Agent Configuration on the Elasticsearch APM currently does NOT support AWS Lambda.
+:::
+
+
+## Prerequisites [_prerequisites]
+
+You need an APM Server to send APM data to. Follow the [APM Quick start](docs-content://solutions/observability/apps/application-performance-monitoring-apm.md) if you have not set one up yet. For the best-possible performance, we recommend setting up APM on {{ecloud}} in the same AWS region as your AWS Lambda functions.
+
+## Step 1: Add the APM Layers to your Lambda function [add_the_apm_layers_to_your_lambda_function]
+
+Both the [{{apm-lambda-ext}}](apm-aws-lambda://reference/index.md) and the Python APM Agent are added to your Lambda function as [AWS Lambda Layers](https://docs.aws.amazon.com/lambda/latest/dg/invocation-layers.html). Therefore, you need to add the corresponding Layer ARNs (identifiers) to your Lambda function.
+
+:::::::{tab-set}
+
+::::::{tab-item} AWS Web Console
+To add the layers to your Lambda function through the AWS Management Console:
+
+1. Navigate to your function in the AWS Management Console
+2. Scroll to the Layers section and click the *Add a layer* button ![image of layer configuration section in AWS Console](../images/config-layer.png "")
+3. Choose the *Specify an ARN* radio button
+4. Copy and paste the following ARNs of the {{apm-lambda-ext}} layer and the APM agent layer in the *Specify an ARN* text input:
+    * APM Extension layer:
+      ```
+      arn:aws:lambda:{AWS_REGION}:267093732750:layer:elastic-apm-extension-{{apm-lambda-ext-v}}-{ARCHITECTURE}:1 <1>
+      ```
+      1. Replace `{AWS_REGION}` with the AWS region of your Lambda function and `{ARCHITECTURE}` with its architecture.
+
+    * APM agent layer:
+      ```
+      arn:aws:lambda:{AWS_REGION}:267093732750:layer:elastic-apm-python-{{apm-python-v}}:1 <1>
+      ```
+      1. Replace `{AWS_REGION}` with the AWS region of your Lambda function.
+
+    ![image of choosing a layer in AWS Console](../images/choose-a-layer.png "")
+5. Click the *Add* button
+::::::
+
+::::::{tab-item} AWS CLI
+To add the Layer ARNs of the {{apm-lambda-ext}} and the APM agent through the AWS command line interface execute the following command:
+
+```bash
+aws lambda update-function-configuration --function-name yourLambdaFunctionName \
+--layers arn:aws:lambda:{AWS_REGION}:267093732750:layer:elastic-apm-extension-{{apm-lambda-ext-v}}-{ARCHITECTURE}:1 \ <1>
```

Comment:
This second line was like this before:

```    --environment "Variables={AWS_LAMBDA_EXEC_WRAPPER=/opt/python/bin/elasticapm-lambda,ELASTIC_APM_LAMBDA_APM_SERVER=<YOUR-APM-SERVER-URL>,ELASTIC_APM_SECRET_TOKEN=<YOUR-APM-SECRET-TOKEN>,ELASTIC_APM_SEND_STRATEGY=background}" ```

Reply:
That's not what I see in the [current live docs](https://www.elastic.co/guide/en/apm/agent/python/current/lambda-support.html#_step_2_add_the_apm_layers_to_your_lambda_function) (switch to `AWS CLI`). I think the change you suggested is actually used in the [next step](https://www.elastic.co/guide/en/apm/agent/python/current/lambda-support.html#_step_3_configure_apm_on_aws_lambda).

Reply:
You are right, I think I mixed different steps :sweat: